b2sdk/raw_api.py

```diff
@@ -11,6 +11,7 @@
 from __future__ import print_function
 
 import base64
+import io
```

Comment:
changes to this file can easily be extracted to another commit/PR that will precede the rest of the changes

Reply:
it is really small change - and yes - quite independent - but I really don't see much value in extracting this to another PR - and also I don't see any value in splitting PR to smaller commits - there is value for not squashing commits during onging review - but I think that squashing before merge is very tempting - I use `blame` a lot, and small commits that fixes small bug after large change sometimes cause like 3-4 backwards steps before I can see real cause for a change...

Reply:
I've just seen that github allows to filter PR diff by commit - but it really works for such independent changes - and I think one should not infer such things when reviewing - what github can do better is to allow review single file (gerrit style) - if you would be able to just see a file list and open each file "inline" or in another tab - then it would be much better experience...